inbm/fpm/inb-provision-cloud/main.go

```diff
@@ -37,6 +36,28 @@ func main() {
 	flag.Usage = usage
 	flag.Parse()
 
+	var uccClientIdFile string
+	var uccServerIdFile string
+	var cloudCredentialPublicDir string
+	var cloudCredentialSecretDir string
+
+	if runtime.GOOS == "windows" {
+		uccClientIdFile = "c:\\ucc\\etc\\client_id"
```

Comment:
Maybe drop the "etc" in the windows path, i.e. c:\ucc\client_id and c:\ucc\server_id

Reply:
> Maybe drop the "etc" in the windows path, i.e. c:\ucc\client_id and c:\ucc\server_id

good idea.. i've changed it.